src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -92,6 +93,11 @@ def load_arguments(self, _):
                    help="Create your Azure Spring Cloud service in an Azure availability zone or not, "
                         "this could only be supported in several regions at the moment.",
                    default=False, is_preview=True)
+        c.argument('build_pool_size',
+                   arg_type=get_enum_type(['S1', 'S2', 'S3', 'S4', 'S5']),
```

Comment:
Could we get the valid value of `build_pool_size` from Python SDK?

Reply:
The valid value of `build_pool_size` is not defined in Python SDK.

Reply:
OK~